modules/template-native/.travis.yml

```diff
@@ -0,0 +1,59 @@
+language: cpp
+
+notifications:
+  email: false
+  
+branches:
+    only:
+        - master
+        - release
+
+addons:
+    apt_packages:
+        - lib32bz2-1.0
+        - lib32ncurses5
+        - lib32z1 
+    apt:
+        sources:
+            - ubuntu-toolchain-r-test
+            - llvm-toolchain-precise 
+        packages:
+            - gcc-7
+            - g++-7
+        
+     
+
+before_script:
+    - wget https://developer.arm.com/-/media/Files/downloads/gnu-rm/8-2018q4/gcc-arm-none-eabi-8-2018-q4-major-linux.tar.bz2? -O /tmp/gcc-arm-none-eabi.tar.bz2
+    - tar -xvjf /tmp/gcc-arm-none-eabi.tar.bz2
+    - export PATH=$PATH:$PWD/gcc-arm-none-eabi-8-2018-q4-major/bin/ 
+    - g++ -v
+    - gcc -v
+    - arm-none-eabi-g++ -v
+
+install:
+    - cd ..
+    - git clone --branch test_whithout_windows.h https://github.com/R2D2-2019/R2D2-build.git 
```

Comment:
same here

Reply:
correct should work i will test it tomorrow and if it works i will change it